packages/mobile/src/account/Support.tsx

```diff
@@ -0,0 +1,76 @@
+import Link from '@celo/react-components/components/Link'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Linking, StyleSheet, Text, View } from 'react-native'
+import SafeAreaView from 'react-native-safe-area-view'
+import SettingsItem from 'src/account/SettingsItem'
+import i18n, { Namespaces } from 'src/i18n'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import Logger from 'src/utils/Logger'
+
+const openExternalLink = (link: string) => {
+  return async () => {
+    try {
+      await Linking.openURL(link)
+    } catch (e) {
+      Logger.error('Support/openExternalLink', '', e)
+    }
+  }
+}
```

Comment:
We already have a helper for this https://github.com/celo-org/celo-monorepo/blob/cc4cb8135bd3765968ca90a7509bef697463a9e3/packages/mobile/src/utils/linking.ts#L16-L32